src/main/com/example3/ArrayRNG.java

```diff
@@ -0,0 +1,37 @@
+package main.com.example3;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class ArrayRNG {
+
+    public static void main(String[] args) {
+
+        int array[] = new int[49];
+        int array2[] = new int[6];
+        array = fillArray(false,array);
+        Arrays.stream(array).boxed().forEach(result -> System.out.print(result  + " "));
+        System.out.println();
+        array2 = fillArray(true, array2);
+        Arrays.stream(array2).boxed().forEach(result -> System.out.print(result + " "));
+
+    }
+
+    static int[] fillArray(boolean fillRandom, int[] array){
+
+        for(int i = 0; i < array.length; i ++){
+            if(fillRandom == false) {
```

Comment:
Podminku pro boolean v ifu se nemusi psat true false 
jestli chces zkontrolovat jestli fillrandom je true .. tak to napises jen if(fillrandom) 
pro false je to s vzkricnikem ... if(!fillRandom)

Reply:
Jasně, to máme stejně, ale někdy to pro přehlednost píšu celé. Už je to opraveno